graph2tac/loader/predict_server.py

```diff
@@ -17,6 +17,8 @@
 import yaml
 os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
 
+import tensorflow as tf
```

Comment:
I just added tensorflow at the top as an import.  It is already being imported by `graph2tac.loader.data_classes`, and now it is needed for the profiler.